.github/workflows/CICD.yml

```diff
@@ -285,6 +285,7 @@ jobs:
         # Autocompletion files
         cp 'target/${{ matrix.job.target }}/release/build/${{ env.PROJECT_NAME }}'-*/out/assets/completions/bat.bash "$ARCHIVE_DIR/autocomplete/${{ env.PROJECT_NAME }}.bash"
         cp 'target/${{ matrix.job.target }}/release/build/${{ env.PROJECT_NAME }}'-*/out/assets/completions/bat.fish "$ARCHIVE_DIR/autocomplete/${{ env.PROJECT_NAME }}.fish"
+        cp 'target/${{ matrix.job.target }}/release/build/${{ env.PROJECT_NAME }}'-*/out/assets/completions/bat.ps1 "$ARCHIVE_DIR/autocomplete/_${{ env.PROJECT_NAME }}.ps1"
```

Comment:
Double-check: Should the `_` really be here? That makes the file be packaged as `_bat.ps1`

Reply:
It is intentional. I think it's a convention for PowerShell scripts - for example see here https://github.com/sharkdp/fd/blob/3ba90dd7685a73923a3a872c4ffc8756a198d755/.github/workflows/CICD.yml#L189

Reply:
I think I should add that underscore in build.rs itself, what do you think?

Reply:
Yes, if the convention is to prefix with `_`, I agree it makes sense to let that be the output of `build.rs` :+1:

Reply:
I would go as far as to say the input file should be called `_bat.ps1.in`

Reply:
Done